src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  age,
+  name,
+  partnerName,
+  sex,
+}) => {
+  let personPartner = 'I am not married';
+
+  if (partnerName) {
+    personPartner = sex === 'f' ? `My husband's name is ${partnerName}`
+      : `My wife's name is ${partnerName}`;
+  }
```

Comment:
Kinda weird logic if you already have this prop: `isMarried`
What if the person have a partner, but they are not married(gf/bf)?

Reply:
ahahha! It's my wierd logic! I want to find bicycle)) But smth went wrong